data/contributors/people.yml

```diff
@@ -2,117 +2,176 @@ name:
 members:
   - name: Levi Naden
     url: https://molssi.org/
+    logo: /images/molssi_logo_mini.png
```

Comment:
Instead of specifying an image path for each person, what about having a `data/organizations/` folder, where you can associate a simple key (e.g. `molssi`) with an image path (local file or remote URL), and then just have `organization: molssi` for each individual?

If the organization key is not found, you just render a blank logo.

Also I *highly* recommend having a `data/contributors/people/` directory so each entry is a separate YAML file.

Reply:
> Also I highly recommend having a data/contributors/people/ directory so each entry is a separate YAML file.

I'm a bit confused on what you are suggesting here. The YAML file is already in the `data/contributors/people`? Are you saying each person should be their own YAML file?